src/index.html

```diff
@@ -32,14 +32,24 @@
 		autoplay: true,
 		// set to true if you want to have video initially muted
 		mute: true,
+		captions: {
+			// set to true if you want video to have captions by default
+			enabled: true,
+			// custom CSS for captions, not all properties are available
+			// see: https://developer.jwplayer.com/jw-player/docs/developer-guide/customization/configuration-reference/#captions
+			styles: null
+		},
 		settings: {
 			// set to true when you want to give user option to enable/disable autoplay
 			// autoplay toggle appears in settings menu which sends event `autoplayToggle` on click
 			showAutoplayToggle: true,
 			// set to true when you want to give user option to change quality of the video
 			// show quality option doesn't appear in Safari and mobile browsers even if the option is set to true
 			// these browsers have native HLS support
-			showQuality: true
+			showQuality: true,
+			// set to true when you want to show captions toggle
+			// captions toggle appears in settings menu, sends event `captionsSelected` on click
+			showCaptionsToggle: true
```

Comment:
It won't be just a toggle soon so I'd rename it to `showCaptions`

Reply:
note for renaming this in specific PR